lib/default-theme/Layout.vue

```diff
@@ -111,6 +112,8 @@ export default {
     this.$watch('$page', updateMeta)
     updateMeta()
 
+    window.addEventListener('scroll', this.onScroll)
```

Comment:
Thank you. come back and see, I really shouldn't bring my feelings into it, and should be more neutral. I'm sorry.
For above problem, the scroll event should be handled in `mount` or `beforeMount` hook, since we are building a SSR application which has some browser api access restrictions.

Reply:
Opps, I must have misread previously.

Tried to removed this whole thread but turns out only be able to remove my own comment. I was not asking a contributor to access `window` at `created()`. Just genuinely curious why he put it in `mounted()` as I obviously just got messed up in my head about the order of `created()` and `mounted()` somehow. Thanks @ulivz for clarifying this.

Reply:
I feel really sorry to make you feel that I am offending you all the time. In case you feel the same again, please do tell me and perhaps suggest a way that I should have put it. 

🎉

Reply:
Work hard together to make this project better. Come on! 🎉